dmapiclient/data.py

```diff
@@ -13,21 +13,23 @@ def init_app(self, app):
 
     def find_audit_events(
             self,
-            audit_type=None,
+            acknowledged=None,
             audit_date=None,
+            audit_type=None,
+            data_supplier_id=None,
+            earliest_for_each_object=None,
+            latest_first=None,
+            object_id=None,
+            object_type=None,
             page=None,
             per_page=None,
-            acknowledged=None,
-            object_type=None,
-            object_id=None,
-            latest_first=None,
-            earliest_for_each_object=None,
```

Comment:
I've been told off for this before because technically changing argument order is an api-breaking change.

Reply:
Agreed - let's keep the existing order for now.

Reply:
Okey dokey, all done